src/ring_theory/polynomial/hermite.lean

```diff
@@ -50,4 +51,69 @@ begin
   simp only [map_one, mul_one, derivative_one, sub_zero]
 end
 
+section coeff
```

Comment:
```suggestion
/-! ### Lemmas about `polynomial.coeff` -/

section coeff
```

`/-!` comments show up in the docs, and `###` is the heading level we usually use

Reply:
Feel free to write a little bit more here if you like, it can span multiple lines!